dhis-2/dhis-api/src/main/java/org/hisp/dhis/audit/TrackerAuditType.java

```diff
@@ -25,34 +25,16 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package org.hisp.dhis.tracker.export.trackedentity;
-
-import java.util.ArrayList;
-import java.util.List;
-import lombok.Data;
-import lombok.experimental.Accessors;
-import org.hisp.dhis.changelog.ChangeLogType;
-import org.hisp.dhis.common.Pager;
-import org.hisp.dhis.trackedentity.TrackedEntity;
-import org.hisp.dhis.trackedentity.TrackedEntityAttribute;
+package org.hisp.dhis.audit;
 
 /**
- * Encapsulation of a web API request for tracked entity data value audit records.
- *
- * @author Lars Helge Overland
+ * @author Halvdan Hoem Grelland
+ * @author Luca Cambi split Changelog from Audit
  */
-@Data
-@Accessors(chain = true)
-public class TrackedEntityAttributeValueChangeLogQueryParams {
-  private List<TrackedEntityAttribute> trackedEntityAttributes = new ArrayList<>();
-
-  private List<TrackedEntity> trackedEntities = new ArrayList<>();
-
-  private List<ChangeLogType> auditTypes = new ArrayList<>();
-
-  private Pager pager;
-
-  public boolean hasPager() {
-    return pager != null;
-  }
+public enum TrackerAuditType {
```

Comment:
Let us find a generic name for this. AuditOperationType ? AuditOperation? Something else?